examples/ConsoleAppWithFailOver/ConsoleAppWithFailOver.csproj

```diff
@@ -6,7 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Azure.Identity" Version="1.5.0" />
+    <PackageReference Include="Azure.Identity" Version="1.7.0" />
```

Comment:
Does this need to be updated as part of this PR?

Reply:
It has some compile errors if I don't add it but it doesn't seem like it happens on preview branch without my changes. That's why I wasn't really sure if I should make a separate PR.